### ISSUE TITLE
global control system volume stopped working 

### ISSUE THREAD

**Issue by beastlyfern** [SHWotever/SimHub#1610]
**Describe the bug**
The global control for system volume is not working. It is still bound to the correct control and I see the volume bar changing on my simhub
[SimHub.txt](https://github.com/SHWotever/SimHub/files/15269761/SimHub.txt)
 dash board but the windows system volume does not change with it.

**Simhub version**
Make sure to run the latest version
tried installing the latest 9.2.15 butt the problem still exists 

**Third party plugins**
Make sure to disable first any third party plugins (It's a critical step before reporting any freeze/performance/crash issues)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Log files**
Join log files available in C:\Program Files (x86)\SimHub\Logs
Note : Bug reports not joining proper log files won't be processed (Never join logs screenshots/photos/extracts, please exclusively join complete logs files).

How to attach files : https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

**Additional context**

**Comment by SHWotever** [comment]
Hi !

If you can still see the volume changing I would guess it's simply because the default sound card has been changed, maybe after swapping some usb ports with the usb sound card for bass shakers.

Can you check in windows sound settings the current default sound card ? 
![image](https://github.com/SHWotever/SimHub/assets/2207331/41a84cc5-21dd-4336-8bc0-075b056c4ee9)

**Comment by beastlyfern** [comment]
Hi,

Yeah for some reason even with my default audio set to the usb dac for my
headphones simhub still only changes my sound card audio levels.

Should use 3-renonessence lab concero but instead it changes Speakers
Realtek usb audio.

The default is always set to my headphones and I don't think any updates
happen so not sure why this started happening all of a sudden.

If you have any ideas it would be greatly appreciated. Thanks so much for
your amazing program.

**Comment by beastlyfern** [comment]
@SHWotever 
Yeah for some reason even with my default audio set to the usb dac for my
headphones simhub still only changes my sound card audio levels.

Should use 3-renonessence lab concero but instead it changes Speakers
Realtek usb audio.

The default is always set to my headphones and I don't think any updates
happen so not sure why this started happening all of a sudden.

**Comment by SHWotever** [comment]
Hi ! I took another look and looks like buried inside windows there is another concept that superseeds the basic defaut sound card concept. I was retrieving the default one based on the "multimedia" role. But I guess this specific sound card has this role assigned. I've switched to console which should represent better the default sound card as seen in windows for the next version.

For reference : https://learn.microsoft.com/en-us/windows/win32/coreaudio/device-roles-in-windows-vista#automatic-device-role-detection)
https://learn.microsoft.com/en-us/windows/win32/api/mmdeviceapi/ne-mmdeviceapi-erole

![image](https://github.com/SHWotever/SimHub/assets/2207331/48c15a8d-fcf4-47d4-9d4d-ce872722e871)

**Comment by beastlyfern** [comment]
@SHWotever Thanks, just to confirm this change should be added to the update after 9.3.0? or is it 9.3.0, if it was added already I still have the same behaviour in 9.3.0. Thanks so much
